conda/plugins/hookspec.py

```diff
@@ -123,3 +129,28 @@ def conda_pre_commands():
                    run_for={"install", "create"},
                )
         """
+
+    @_hookspec
+    def conda_post_commands(self) -> Iterable[CondaPostCommand]:
+        """
+        Register post-commands functions in conda.
+
+        **Example:**
+
+        .. code-block:: python
+
+           from conda import plugins
+
+
+           def example_post_command(command, args):
+               print("post-command action")
```

Comment:
```suggestion
           def example_post_command(command, args, result):
               print("post-command action")
```

Reply:
Here's what I meant in our meeting on Monday, I know this is mostly copy-pasting the pre_command code, but thought I catch this early. The post command should also be given the result (error code? subcommand return value?) of the command that was run.